frontend/projects/valtimo/components/src/lib/components/widget-layout-info/widget-layout-info.component.html

```
@@ -0,0 +1,9 @@
+<cds-inline-notification
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)